### DISCUSSION TITLE
Dashboard doesn't load

### DISCUSSION THREAD

**Discussion by Zoddo** [argoproj/argo-rollouts#3754] [Q&A] [answered]
Hello,

Currently trying Argo Rollouts, I'm having issue to make the dashboard to work.

When I execute `kubectl argo rollouts dashboard`, I get the following ouput:
```
INFO[0000] Argo Rollouts Dashboard is now available at http://localhost:3100/rollouts 
INFO[0000] [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {Addr: "0.0.0.0:3100", ServerName: "0.0.0.0:3100", }. Err: connection error: desc = "transport: Error while dialing: dial tcp 0.0.0.0:3100: connect: connection refused"
```

and when opening the dashboard, it's stuck at the Loading screen.

For context, I'm working on a remote k8s cluster, and I execute the kubectl command locally on my workstation. Can you confirm if this is supported, or if the dashboard is only meant for test clusters than run on the local workstation?

**Comment by adrien-barret** [comment]
Same issue exactly. would say yes it's supported. 
the command `Argo Rollouts Dashboard` seem to just be a port-forward anyway.

**Comment by Zoddo** [comment, marked as answer]
Ok, so I came back on this issue again. My issue was that I didn't create any rollout yet (since I tried to spawn the dashboard before following the Getting Started docs). After creating a rollout, the dashboard worked correctly.

The `failed to connect` error wasn't related to my issue at all, I still have it and the dashboard works correctly.

It may be a good idea to display something like "There is no rollout" instead of being stuck on the loading screen, thinking something is broken?

**Reply by kostis-codefresh** [reply]
Could you please open a GitHub issue for this?

**Comment by adrien-barret** [comment]
solved my issue by installing rollout extension directly in argocd UI with argo extension. it add a tab "rollout" on every app when you clic on app's detail.
